### DISCUSSION TITLE
Unsupported MIME type: image/webp

### DISCUSSION THREAD

**Discussion by Xaypanya** [oliver-moran/jimp#1212] [Q&A] [answered]
I got this error message "Unsupported MIME type: image/webp". How to fix this?

**Comment by hipstersmoothie** [comment, marked as answer]
Currently don't support web-p. We do plan on it in the future but you could do it all yourself by:

1. writing a plugin like png,jpeg
2. Using a wasm webp implementation

It should be possible, but we probably won't do this for awhile

**Reply by Xaypanya** [reply]
It works, thank you. I hope Jimp will support WebP image in the future.

**Reply by hipstersmoothie** [reply]
If you did have code for that feel free to submit a pr :D

**Comment by wsGolden** [comment]
I didn't use the web-p format file and still encountered this issue
